server/protocols/jmap/src/main/java/org/apache/james/jmap/JMAPRoutes.java

```diff
@@ -27,13 +27,32 @@
 
 import org.reactivestreams.Publisher;
 import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import reactor.core.publisher.Mono;
 import reactor.netty.http.server.HttpServerRequest;
 import reactor.netty.http.server.HttpServerResponse;
 import reactor.netty.http.server.HttpServerRoutes;
 
 public interface JMAPRoutes {
+    class ResponseUtil {
+        private static final Logger LOGGER = LoggerFactory.getLogger(ResponseUtil.class);
```

Comment:
you loose the context by doing so (ResponseUtil is not very helpful to track problems).
What about just passing a logger as a parameter?